### DISCUSSION TITLE
Türkiyenin Genel Afet Bölgeleri ve Oluşabilecek Deprem Büyüklüklerinin Görüntülenebilmesi

### DISCUSSION THREAD

**Discussion by agitrubard** [acikkaynak/deprem-yardim-projesi#82] [Ideas]
Yer Bilimciler ile istişare ederek, hangi bölgelerde kaç büyüklüğünde deprem olabileceği ve hangi alanların ne kadar etkilenebileceği konusunda bilgilendirme amaçlı bir ekran oluşturulabilir. Genel anlamda Yer Bilimciler deprem ile ilgili belirsiz olan tek konunun; "depremin olacağı zaman", diyorlar. Buradan yola çıkarsak diğer verileri işleyerek bu konuyla ilgili farkındalık oluşturabiliriz. Aksiyon alınması gereken bölgelerle ilgili de bilgi verebiliyor oluruz.

**Comment by sercangoger** [comment]
evet bir sürü farklı haritalar mevcut buradaki afet haritalarının datalarını alıp projeler geliştirmek güzel olacaktır.
